app/routes/ready-reckoner/selected-summary.js

```diff
@@ -0,0 +1,109 @@
+const pageDetails = {
+  path: '/selected-summary',
+  nextPath: '/',
+  template: 'selected-summary'
+}
+
+const textMapping = {
+  arable: 'Arable land',
+  grassland: 'Grassland',
+  hedgerow: 'Hedgerows'
+}
+
+function getContentDetails (calculation, selectedStandards) {
+  let totalPayment = 0
+  let optionHtml = ''
+
+  selectedStandards.forEach(s => {
+    totalPayment += calculation[s].payment
+    optionHtml += textMapping[s] + '<br> '
+  })
+
+  return {
+    title: 'Summary',
+    components: {
+      insetText: {
+        text: `Based on your information and options, you would receive at least £${totalPayment.toFixed(2)} a year, paid in 12 monthly installments of £${(totalPayment / 12.0).toFixed(2)}.`
```

Comment:
Do you think it would be worth creating a ticket to cover some work on adding payment options during the calculation to avoid having to do this here?

Reply:
Definitely.